lasagne/updates.py

```diff
@@ -20,6 +20,13 @@
 * :func:`apply_momentum()`
 * :func:`apply_nesterov_momentum()`
 
+We additionally provide a function to scale the norm of a list of tensors:
+
+* :func:`step_scaling()`
+
+Step scaling can be used to scale the gradient steps when training recurrent
+neural networks.
+
```

Comment:
You should have a look at the full diff yourself in between ;) I'd just add that to the next paragraph instead ("of a tensor variable or multiple tensor variables:")